app/routes/invitations.rb

```diff
@@ -0,0 +1,76 @@
+module ExercismWeb
+  module Routes
+    class Invitations < Core
+      post '/teams/:slug/invitation' do |slug|
+        please_login
+        only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
+          team.invite_with_usernames(params[:usernames], current_user)
+          team.save
+
+          flash[:success] = "Invitation sent to team #{slug}"
+          redirect "/teams/#{slug}/manage"
+        end
+      end
+
+      post '/teams/:slug/invitation/accept' do |slug|
```

Comment:
This should also be `invitations` plural.

I am still uneasy about the `accept` verb in the route, but I think we'll just live with it for a while until I can step back for a bit and think about all the routes more clearly.